### DISCUSSION TITLE
Please tell me your favorite Game!

### DISCUSSION THREAD

**Discussion by ikeno0zz7** [NAIST-SE/AttractingContributors#1183] [2024 Attraction Contributions]
please click here!
https://github.com/ikeno0zz7/Game

**Comment by Idescrew** [comment]
I told me my favorite game.
Could you tell me Japanese summer festivals you would like to visit.

https://github.com/Idescrew/AttractingContributors

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel

**Comment by momoka-jp** [comment]
https://github.com/momoka-jp/Place

Please indicate your favourite places to eat near NAIST.

**Comment by masataka-ki** [comment]
I've sent you a pull request, so if you want, please pull request for me too!
https://github.com/masataka-ki/AttractingContributors-Please-tell-me-a-country-you-want-to-visit-in-the-future
